src/Testing/BaseMutatorTestCase.php

```diff
@@ -189,7 +195,7 @@ private function assertSyntaxIsValid(string $realMutatedCode): void
             $returnCode,
             sprintf(
                 'Mutator %s produces invalid code',
```

Comment:
```suggestion
                'The mutator "%s" produced invalid PHP code.',
```

Might be worth tweaking it to print for which output this is the case?